src/pages/talks.js

```diff
@@ -0,0 +1,90 @@
+// By Matt
+
+/** @jsx jsx */
+
+import { Container, jsx, Styled } from 'theme-ui';
+import { graphql } from 'gatsby';
+import React from 'react';
+import { Seo } from '../components/seo';
+import { Layout } from '../components/layout';
+import { Cast, Code, PlayCircle } from 'react-feather';
+import { IconLink } from '../components/icon-link';
+
+// Materials component
+function TalkMaterial({ type, url }) {
```

Comment:
Don't copy paste. Since `ScheduleItem` use the similar things, pull them out into separate component, maybe call it `TalkMaterialIconLink`.